Sources/macro.js

```diff
@@ -185,7 +187,7 @@ export function obj(publicAPI = {}, model = {}) {
       return;
     }
 
-    if (otherMTime && otherMTime < model.mtime) {
+    if (otherMTime && otherMTime < publicAPI.getMTime()) {
```

Comment:
not good keep the old one

Reply:
Maybe not... It is fine.

Reply:
we should always read mtime using getMTime, shouldn't we?

Reply:
yes